scripts/publish.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const fs = require('fs');
+const { execSync } = require('child_process');
+const { getPackageJSON } = require('./helpers');
+const COMPONENTS_PATH = path.join(__dirname, '../components');
+
+function getPublicVersion(package) {
+    return execSync(`npm view ${package} version`, {encoding: 'utf8'}).replace('\n', '');
+}
+
+function hasAnUpdate(component) {

```

Comment:
```suggestion
function shouldUpdate(component) {
```
Rename in the `main()` also.

Reply:
Yes, this name is a lot better.
Changed everywhere.